### ISSUE TITLE
Why did you replace the SGD with AdamW?

### ISSUE THREAD

**Issue by HansolEom** [signatrix/efficientdet#13]
I saw that SGD was used in the paper. 
So why did you use AdamW? 
Were there any performance differences?

**Comment by uvipen** [comment]
There are 2 reasons:

- First, we did try using SGD as mentioned in the paper, but the loss did not converge very well during warm-up training steps
- Second, from the paper, it is not very clear how many iters each model was trained (We think ~ 120k iters, based on Figure 5), Then it would be hard for you to define what is 5% of training process.

We are training and fine-tuning other params as well, not only optimizer

**Comment by HansolEom** [comment]
Thank you for your reply.

**Comment by TangShengqin** [comment]
question +１
